ui/settings_page.go

```diff
@@ -252,7 +276,7 @@ func (pg *settingsPage) agent() layout.Widget {
 func (pg *settingsPage) mainSection(gtx layout.Context, title string, body layout.Widget) layout.Dimensions {
 	return layout.Inset{Bottom: values.MarginPadding10}.Layout(gtx, func(gtx C) D {
 		return pg.theme.Card().Layout(gtx, func(gtx C) D {
-			return layout.UniformInset(values.MarginPadding15).Layout(gtx, func(gtx C) D {
+			return layout.Inset{Top: values.MarginPadding15, Left: values.MarginPadding15, Right: values.MarginPadding15}.Layout(gtx, func(gtx C) D {
```

Comment:
MarginPadding15 can be assigned to a variable to break the code to the next line.

Reply:
i'm not sure what you mean here?